crates/bevy_ecs/src/query/mod.rs

```diff
@@ -1,3 +1,14 @@
+//! Helper types for [`Query`](crate::system::Query).
```

Comment:
I try to avoiding using the word "helper" in documentation and code because it is too generic. You could write this same line for literally anything.

Imo this module doc should instead describe queries from a high level. If that seems redundant with the Query type docs, feel free to just omit the module doc entirely.